lox_services/persistence/database/utils.py

```diff
@@ -2,9 +2,11 @@
 import os
 import re
 from datetime import datetime, timedelta, timezone
-from typing import Literal
+from typing import Callable, Literal, Sequence, Union
 
+from tabulate import tabulate
 import pandas as pd
+import pycountry
```

Comment:
Add to requirements.txt

Reply:
requirements.txt, line 36 - `pycountry >= 22.3.5`